docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -59,6 +59,16 @@ C:\>where bash
 C:\Program Files\Git\bin\bash.exe
 ```
 
+Special Note: If you have WSL installed on your platform, it will most likely show up as the first item as
+c:\Windows\System32\bash.exe. You can delete WSL in your platform if possible OR rename this to bashWSL.exe.
```

Comment:
I would highly recommend not suggesting this. We should (1) figure out why this doesn't work with WSL Bash (probably just a missing dependency?) and (2) instead suggest changing the PATH temporarily to use Git Bash instead.

Reply:
I already filed a bug for the WSL conflict (https://github.com/Microsoft/openenclave/issues/1302) and will reference that here. This is a workaround until the conflict has been resolved and/or we support WSL bash. We can build OE in a WSL bash window but cross-compiling in a Native Command Tools Window is a different environment.
You cannot change the PATH to use Git Bash as this is the system path and Windows doesn't work.
Will update this and resend. We can have the suggested workaround in our wiki page.

Reply:
It is better to not mention this workaround. 
A better recommendation is to reorder the paths by editing the PATH variable.

Reply:
Sorry I cannot recommend reordering the paths as that did not work for me. Will just mention the conflict.